openstack/cronus/templates/maillog/role.yaml

```diff
@@ -0,0 +1,17 @@
+{{- if .Values.maillog.enabled -}}
+apiVersion: rbac.authorization.k8s.io/v1
```

Comment:
Are you sure this is needed? I think by default it should be able to get all secrets in its own namespace